mhapsite/mhap/models.py

```diff
@@ -45,6 +45,7 @@ def get_list_url(self):
 #http://stackoverflow.com/questions/1737017/django-auto-now-and-auto-now-add
 class ChatMessages(models.Model):
     message = models.CharField(max_length=255)
+    message_type = models.CharField(max_length=255, default="default")
```

Comment:
It needs to be a char field either way, right? This is where I am right now. Is this at all close to what you're thinking?
`choices = (("default", "default"),
                 ("resource", "resource"))`
`message_type = models.CharField(max_length=10, choices=choices, default="default")`